lib/routes/user/home/payment_item.dart

```diff
@@ -15,6 +15,9 @@ class PaymentItem extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     String title = _paymentInfo.title;
+    if (title.contains("Bitrefill")) {
```

Comment:
Better to use `startsWith` then `contains`

Reply:
Are you sure? I'd keep contains

Reply:
It is a temporary solution and my thinking is to provide mistakes here (if someone enters Bitrefill in the text...).
Anyway If we want to be more "loose" here then we need to change the next line as well to:
`title = "Bitrefill"` instead of `title=title.substring(0,10)`

Reply:
I'm good either way

Reply:
> It is a temporary solution and my thinking is to provide mistakes here (if someone enters Bitrefill in the text...).
> Anyway If we want to be more "loose" here then we need to change the next line as well to:
> `title = "Bitrefill"` instead of `title=title.substring(0,10)`
I've replaced contains with startsWith and set title to "Bitrefill".

Reply:
Did a full pass over the code and since all the changes are regarding the description (title) and image I believe a better strategy is to do these two small changes in the PaymentInfo model itself. We can add these manipulations inside both `imageURL` and `description` getters and I think all these changes in the UI won't be needed.
Maybe a small change will need to be done in BreezAvatar.
After all when we will do the generic solution the usage will be like this (coming from the model) and we won't need to change the UI code.
What do you think @erdemyerebasmaz ?

Reply:
Yes, it makes sense to move data manipulations out of UI level.
I've moved them to PaymentInfo model and added _VendorAvatar to BreezAvatar for vendors.